CHANGELOG.md

```diff
@@ -29,6 +29,12 @@ The general format is:
 
 ```
 
+# 1.1.0 - DATE (13/01/2022)
+
+### Added
+
+- add support for the timezone attribute of CronWorkflow and validate the specified timezone
```

Comment:
What do you think about including a small note about `pytz` so we know when it was introduced? 

```suggestion
- add support for the timezone attribute of CronWorkflow and validate the specified timezone
- introduce `pytz` dependency for timezone validation
```